pages/templates/pages/candidate_endorsement_index_page.html

```diff
@@ -55,20 +55,54 @@ <h3 class="pull-left mb0">
       <div id="candidates-list">
         <div class="no-result text-center"><h3>No Matching Candidates Found</h3></div>
         <ul class="card-list list">
-          {% for candidate_page in candidates %}
-              {% with candidate_page.candidateendorsementpage.candidate as candidate %}
-                  <li class="card clearfix">
-                      <a href="{% pageurl candidate_page %}">
-                          <div class="card__photo" style="background-image: url('{{ candidate.photo.url }}')"></div>
-                          <div class="card__content">
-                              <span class="card__item name">{{ candidate.name }}</span>
-                              <span class="card__item bold smaller state">{{ candidate.state }} {{ candidate.district }}</span>
-                              <span class="card__item bottom">{% if candidate.primary_date %}<div class="card__item smaller bold">Election: <span class="date hidden">{{ candidate.primary_date|date:"U" }}</span>{{ candidate.primary_date }}</div> {% endif %}{{ candidate.office }}</span>
-                          </div>
-                      </a>
-                  </li>
-              {% endwith %}
+
+          {% for candidate in candidates %}
+          {% with candidateendorsementpage=candidate.candidateendorsementpage %}
+
+          <li class="card clearfix">
+
+            <a href="{% pageurl candidateendorsementpage %}">
+
+              {% image candidateendorsementpage.photo original as photo_img %}
```

Comment:
What's reasoning behind `candidateendorsementpage` instead of `candidate_endorsement_page`? Probably need to establish guidelines here, too, but latter seems more readable.

Reply:
Actually I'm a bit confused about the model here, but `candidate.candidateendorsementpage` is something that already exists for some reason. So all I did was shorten it and remove the prefix basically. I'm confused why the page object is somehow nested one level down, since the logic seems like it should be returning a list of page objects?